### PR TITLE
Add prayer times for Qasre

### PR DESCRIPTION
**Pull request by Muhammadbarznji**
**Description:**
This pull request introduces essential updates to incorporate Qasre prayer time information into the database, along with the necessary city mapping. Additionally, relevant tests have been modified or added to ensure the accurate functioning of the system with the newly integrated data.

**Changes Made:**

**Database Update:**
- Qasre prayer time data has been added to the existing database schema. This includes fields for Fajr, Dhuhr, Asr, Maghrib, and Isha prayer times specific to Qasre.

**City Mapping:**
- The necessary city mapping has been implemented to link Qasre with its corresponding geographical coordinates. This ensures precise calculation of prayer times for the specified location.

**Test Updates:**
- Existing tests related to prayer time calculations have been updated to accommodate the inclusion of Qasre. New test cases have been added to verify the accuracy of the data integration and mapping processes.
- New tests for city mapping have been added to ensure that the system correctly associates Qasre with its geographical coordinates.

**Related Issue:**
- [Issue #4](https://github.com/kosratdev/muslim-data-ios/issues/4)

### CODE REVIEW

File: MuslimData/Assets/muslim_db_v1.3.2.db
Comment (kosratdev) [inline MuslimData/Assets/muslim_db_v1.3.2.db:1]: With every change in the database, I prefer to change the DB version (minor or patch). Furthermore, I'm considering creating an MD file in this folder to keep tracking db changes.

**Review by kosratdev — CHANGES_REQUESTED** [review]
Thank you very much for your efforts kaka @Muhammadbarznji. Please just update the database version and I will add the MD file to the Assets folder to keep track the DB changes.

File: MuslimData/Assets/muslim_db_v1.3.2.db
Comment (Muhammadbarznji) [reply MuslimData/Assets/muslim_db_v1.3.2.db:1]: Hi kak @kosratdev, Thanks for your comments! I've addressed the changes you mentioned in the PR. The database version has been successfully updated,

**Review by kosratdev — APPROVED** [review]
Looks good for me. Thank you very much for your efforts 👏

**Pull request merged**